website/docs/reference/notifications.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Unleash's notifications give you updates when certain events occur in projects that you are part of. The notifications are accessible from the notifications button in the navigation section of the admin UI.
+
+Your notifications will only contain updates from other members; you will not get notifications about actions you perform yourself.
+
+![Notifications](/img/notifications.png)
```

Comment:
Oh, I missed this 😅  Please make sure the alt text is useful to people who can't see the image for whatever reason. "Notifications" doesn't really explain much at all. I've added a suggestion below:

```suggestion
![The notifications overview. It's telling the user about a feature that was enabled in production and that there is a change request ready for review.](/img/notifications.png)
```